tasks.py

```diff
@@ -146,16 +144,26 @@ def run_security_tests(*, image: str):
 
 # git
 REPO = git.Repo(CWD)
-COMMIT_HASH = REPO.head.object.hexsha
+COMMIT_HASH = REPO.head.object.hexsha[:7]
```

Comment:
Add support for short hash instead? Unsure if it matter for this size of repo.

Reply:
This is changing the hash into a "short hash" and the number of characters is the git default (although `git` is smart enough to pick a longer hash if it detects a collision, and this is not).

Let me make it smarter

Reply:
It is now smarter and uses [a dynamic config](https://git-scm.com/docs/git-config#Documentation/git-config.txt-coreabbrev) by default